dry_adiabatic_adjust/dadadj.F90

```diff
@@ -50,22 +50,24 @@ end subroutine dadadj_init
   !> \section arg_table_dadadj_run Argument Table
   !! \htmlinclude dadadj_run.html
   subroutine dadadj_run( &
-       ncol, dt, pmid, pint, pdel, state_t, state_q, cappa, t_tend, &
+       ncol, nz, dt, pmid, pint, pdel, state_t, state_q, cappa, cpair, s_tend, &
        q_tend, dadpdf, scheme_name, errmsg, errflg)
 
     !------------------------------------------------
     !   Input / output parameters
     !------------------------------------------------
     integer,  intent(in)        :: ncol  ! number of atmospheric columns
+    integer,  intent(in)        :: nz    ! number of atmospheric levels
     real(kind_phys), intent(in) :: dt          ! physics timestep
     real(kind_phys), intent(in) :: pmid(:,:)   ! pressure at model levels
     real(kind_phys), intent(in) :: pint(:,:)   ! pressure at model interfaces
     real(kind_phys), intent(in) :: pdel(:,:)   ! vertical delta-p
     real(kind_phys), intent(in) :: cappa(:,:)  ! variable Kappa
+    real(kind_phys), intent(in) :: cpair(:,:)  ! heat capacity of air
     real(kind_phys), intent(in) :: state_t(:,:)   ! temperature (K)
     real(kind_phys), intent(in) :: state_q(:,:)   ! specific humidity
-    real(kind_phys), intent(out), target :: t_tend(:,:)   ! temperature tendency
-    real(kind_phys), intent(out), target :: q_tend(:,:)   ! specific humidity tendency
+    real(kind_phys), intent(out) :: s_tend(:,:)   ! temperature tendency
```

Comment:
I would change this comment to state that the tendency is for dry air enthalpy:

```suggestion
    real(kind_phys), intent(out) :: s_tend(:,:)   ! dry air enthalpy tendency
```

Reply:
Fixed comment.